lib/rafka/consumer.rb

```diff
@@ -124,21 +124,34 @@ def consume(timeout=5)
     # If more than one messages refer to the same topic/partition pair,
     # only the largest offset amongst them is committed.
     #
-    # @note This is non-blocking operation; a successful reply means
+    # @note This is non-blocking operation; a successful server reply means
     #   offsets are received by the server and will eventually be committed
     #   to Kafka.
     #
-    # @param [Array<Message>] the messages for which to commit offsets
+    # @param msgs [Array<Message>] the messages for which to commit offsets
     #
     # @raise [ConsumeError] if there was any error commiting offsets
     #
-    # @return [Boolean] whether offsets were committed successfully
+    # @return [Integer] the number of offsets sent to the server
     def commit(*msgs)
+      offsets = []
+      tp = Hash.new { |h, k| h[k] = { partition: nil, offset: 0 } }
```

Comment:
We (should/will) support multiple topics per consumer, with that in mind we have to group by `(topic, partition, offset)`

Reply:
It is supported - `k` is the topic.

Reply:
Nvm my previous comment. Yes it will be supported.